ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -124,7 +124,7 @@
             {{ item.mhrNumber }}
           </template>
           <template v-slot:[`item.state`]="{ item }">
-            {{ item.status }}
+            {{ getOwnerStatus(item.ownerStatus) }}
```

Comment:
Was `status` too  `ownerStatus` a recent property name update done on the BE?

Reply:
@cameron-freshworks I am not sure what BE is, but ownerStatus was the field that should have been used.  The api was returning it.  It is the owner.status.  Ali confirmed it.

Reply:
Sounds good, thanks!